GardenApp/app/src/main/java/com/cs1530_group1/gardenapp/ViewSpeciesInfoActivity.java

```diff
@@ -53,54 +52,67 @@ protected void onCreate(Bundle savedInstanceState) {
             return;  //stop trying, nothing was received.
         }
 
+
+        updateTextViews();
```

Comment:
I would like commenting here. I know the method is right below it, but I want to know what it does before it's called.